src/builder/pod/mod.rs

```diff
@@ -25,8 +27,18 @@ use k8s_openapi::{
 };
 use tracing::warn;
 
+#[derive(Debug, thiserror::Error, PartialEq)]
```

Comment:
```suggestion
#[derive(Debug, thiserror::Error)]
```

Error types are typically not `PartialEq`.

Reply:
Yeah, that's a leftover of my assert_eq!() test, which didn't work because of `TryFromIntError`